nvi-commons/src/main/java/no/sikt/nva/nvi/common/service/model/Candidate.java

```diff
@@ -260,6 +261,18 @@ public boolean isReported() {
         return REPORTED.equals(reportStatus);
     }
 
+    public boolean isNotReportedInClosedPeriod() {
```

Comment:
Is this a model class used in the API? If so, you should consider @Transient or @JsonIgnore on these method to avoid that they are rendered as fields in JSON.

Reply:
No, thats `no.sikt.nva.nvi.common.service.dto.CandidateDto`